example/imx8mm/passive_server/server.c

```diff
@@ -0,0 +1,29 @@
+
+#include <sel4cp.h>
+
+sel4cp_msginfo 
+protected(sel4cp_channel ch, sel4cp_msginfo msginfo)
+{
+    switch (sel4cp_msginfo_get_label(msginfo)) {
+        case 0:
+            sel4cp_dbg_puts("Server is running on clients scheduling context\n");
```

Comment:
similar to comment on the `client` side. Please use `server:` consistently for the printing.